stock_warehouse_flow/models/stock_move.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _action_confirm(self, merge=True, merge_into=False):
+        # Apply the flow configuration on the move before it generates
+        # its chained moves (if any)
+        for move in self:
+            if not move._apply_flow_on_action_confirm():
+                continue
+            flow = self.env["stock.warehouse.flow"]._search_for_move(move)
+            flow._apply_on_move(move)
+        return super()._action_confirm(merge=merge, merge_into=merge_into)
+
+    def _apply_flow_on_action_confirm(self):
+        return True
```

Comment:
Shouldn't we limit that to only deliveries?
```suggestion
        return self.picking_type_id.code == "outgoing"
```

Reply:
Well, when coding the whole module I had in mind that it could also handle reception flows, not only delivery ones (with some new code of course).

Reply:
The from_picking_type_id is restricted currently on "outgoing", so I think it is better to be consistent here